Entities/Common/Controls/StandardControls.as

```diff
@@ -56,7 +56,12 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 		CBlob@ owner = getBlobByNetworkID(params.read_netid());
 		CBlob@ pick = getBlobByNetworkID(params.read_netid());
 
-		if (owner !is null && pick !is null)
+		if (owner !is null 
+		    && !owner.isInInventory()
+		    && !owner.isAttached()
+		    && pick !is null 
+		    && !pick.isAttached()
+		    && !pick.hasAttached())
```

Comment:
!pick.hasAttached() could potentially mess with some mods, so instead use pick.canBePickedUp(). This still works the same with the dinghy.
```suggestion
		    && pick.canBePickedUp(owner))
```